packages/components/src/molecules/ReviewCard/ReviewAuthor.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes } from 'react'
+import Tooltip from '../Tooltip'
+import Icon from '../../atoms/Icon'
+
+export interface ReviewAuthorProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * The author of the review.
+   */
+  author: string
+  /**
+   * Whether the author is verified or not. Defaults to false.
+   */
+  isVerified: boolean
+  /**
+   * Text to be displayed in the tooltip when hovering over the verified icon. Defaults to 'Verified User'.
+   */
+  verifiedText?: string
+}
+
+const ReviewAuthor = forwardRef<HTMLDivElement, ReviewAuthorProps>(
+  function ReviewAuthor(
+    { author, isVerified, verifiedText = 'Verified User', ...otherProps },
+    ref
+  ) {
+    return (
+      <div data-fs-review-card-author {...otherProps} ref={ref}>
+        <span data-fs-review-card-author-name>{author}</span>
+        {isVerified && (
+          <Tooltip content={verifiedText} placement="bottom">
```

Comment:
```suggestion
          <Tooltip content={verifiedText} placement="bottom-center">
```

Reply:
Because of the previous changes 
<img width="640" alt="image" src="https://github.com/user-attachments/assets/b4627d5e-44b7-405b-acb0-0ec9b05d2548" />